Dockerfile.QA

```diff
@@ -45,8 +45,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # image_preprocess custom backend requires opencv
```

Comment:
If we are building custom backend here, does `python3-opencv` satisfy the dependency on C++ side?